doc/redis.md

```diff
@@ -0,0 +1,20 @@
+# Cloud Resource Operator - Redis
+
+## Usage
+To seed a Kubernetes/Openshift cluster with an example Redis resource:
+```
+$ make cluster/prepare 
+$ make cluster/seed/redis
+```
+
+## Strategy 
+A config map object is expected to exist for each provider that will be used by the operator. This config map contains provider-specific information about how to deploy a particular resource type, such as blob storage. In the Cloud Resources Operator, this provider-specific configuration is called a strategy. An example of the AWS strategy can be seen [here](../deploy/examples/config_aws_strat.yaml)
+### AWS Strategy
+A JSON object containing two keys - `region`, which is the AWS region code. `strategy`, which is a JSON representation of [this struct](https://docs.aws.amazon.com/sdk-for-go/api/service/elasticache/#CreateReplicationGroupInput)
+
+### Kubernetes/Openshift Strategy
+For Kubernetes/Openshift a JSON object containing a single key `strategy`, under strategy is the following keys which are used to overwrite the object specs:
```

Comment:
Just a suggestion, my version might not be very clear but I was a bit confused by this sentence when I first read it 
```suggestion
For Kubernetes/Openshift the JSON object contains a single key, `strategy`. The `strategy` object can contain the  following keys which are used to overwrite specific object configuration: 
```